src/Types.jl

```diff
@@ -690,6 +758,9 @@ function handle_repos_add!(ctx::Context, pkgs::AbstractVector{PackageSpec};
             if !folder_already_downloaded
                 version_path = Pkg.Operations.find_installed(pkg.name, pkg.uuid, pkg.repo.git_tree_sha1)
                 mkpath(version_path)
+                @static if Sys.iswindows()
```

Comment:
This was enough to work around the filesystem error

Reply:
Urgh, do we really need it in the Pkg code itself? Can't we put it in the test code? If it really is needed I guess we can keep it here.

Reply:
I think it needs to be here since it is the `mv` call inside of Pkg that triggers the error. Is there a way to work around that from outside of Pkg?

FWIW I'm not thrilled about it either.